### DISCUSSION TITLE
Cant get output log in urat

### DISCUSSION THREAD

**Discussion by changpenghui616** [memtest86plus/memtest86plus#428] [Q&A]
Im use the last version memtest86+ to test，command is ‘memtest.efi console=ttyS0’，but  nothing in serial port. Im sure my PC is normal（cause use debug uefi version to power on，it have log）

**Comment by 01e3** [comment]
Hi.

Could you please clarify how are you starting memtest86+ (UEFI boot loader, EFI Shell, loader like iPXE, GRUB?) and how are you providing the parameters?

**Comment by changpenghui616** [comment]
Sure，I use  flash driver to boot enter the efi shell，and run command ‘memtest.efi console=ttyS0’，the memtest86+ process can run，but no output in serial log

**Reply by x86fr** [reply]
I'm not sure a UEFI shell will pass the parameter correctly (as a linux kernel parameter). Anyway, what motherboard do you use?

**Reply by changpenghui616** [reply]
i try to display the enable_tty value in screen， it is ‘1’，so i think parameter is right. my motherboard  is Meteor Lake，Intel® Core™ Ultra processors (Series 1)

**Reply by x86fr** [reply]
What is the exact model of the motherboard and how is your COM port physically connected?

**Reply by changpenghui616** [reply]
memtest86+ display model is “Intel(R) Core(TM) Ultra 7 155H”.
the motherboard have reserve the urat port to debug，and i use the little board for TTL transfor USB  connect the debug port to host.

**Reply by x86fr** [reply]
What's the exact model of the motherboard? (Not the CPU)

**Reply by changpenghui616** [reply]
its honor MagicBook Art 14
